README.md

```diff
@@ -1,9 +1,70 @@
-# versionize_derive
+**Versionize is a framework for version tolerant serializion/deserialization of Rust data structures, designed for usecases that need fast deserialization times and minimal size overhead. It does not aim to be a generic serialization framework and only the [bincode](https://crates.io/crates/bincode) backend is supported.**
```

Comment:
Lines in markdown documents should not exceed a length of 80 characters. There are more lines that break this rule below as well.

Reply:
fixed it.